src/System.Security.Cryptography.Algorithms/src/System/Security/Cryptography/Rfc2898DeriveBytes.cs

```diff
@@ -249,30 +250,45 @@ private void Initialize()
         }
 
         // This function is defined as follows:
-        // Func (S, i) = HMAC(S || i) | HMAC2(S || i) | ... | HMAC(iterations) (S || i) 
+        // Func (S, i) = HMAC(S || i) ^ HMAC2(S || i) ^ ... ^ HMAC(iterations) (S || i) 
         // where i is the block number.
         private byte[] Func()
         {
             byte[] temp = new byte[_salt.Length + sizeof(uint)];
```

Comment:
It's user-provided and arbitrarily large.  I thought about removing this one, too, but the average number of calls per object lifetime is ~2.  So I went with sanitization's 5-nines instead of sterilizations 9-nines.